PURL-TYPES.rst

```diff
@@ -471,6 +471,18 @@ including container images built by Docker and others:
       pkg:oci/static@sha256%3A244fd47e07d10?repository_url=gcr.io/distroless/static&tag=latest
       pkg:oci/hello-wasm@sha256%3A244fd47e07d10?tag=v1
 
+pear
+----
+``pear`` for PEAR and PECL PHP packages:
+
+- The default repository for PEAR is ``https://pear.php.net``. The default repository fo PECL is ``https://pecl.php.net/``.
+- The ``namespace`` is the channel of the package.
```

Comment:
please elaborate how a namespace is derived from a channel, and which role a repository plays in all of this.

Reply:
This would keep the Purl in line with other ecosystem like go.

Reply:
This approach makes it slightly easier to filter between the PECL and PEAR packages

Reply:
> This would keep the Purl in line with other ecosystem like go.

FYI: the go people are not happy with their current type spec. 
see https://github.com/package-url/purl-spec/issues/308

Reply:
> This approach makes it slightly easier to filter between the PECL and PEAR packages

Do not get me wrong, I fully support your approach to use channels as namespaces, as I stated here: https://github.com/package-url/purl-spec/pull/300#pullrequestreview-2156893477

The thing is: channel names seam to derive from URLs. Per spec/rfc, URL's host-part are lowercased (case-insensitive, and canonical lowercase?).
Therefore, it would be necessary to also lowercase domain names here, as they are used for namespaces here. And this should be stated in the spes of PURL-TYPE.

Reply:
i see this addendum was made via https://github.com/package-url/purl-spec/compare/e2eb71b2cb7ad3e43aa8f2c4412d1a7dc8364a66..6371ff4b92260866a58ced93653b306e518b91f5

Reply:
Consider this being closed. thanks, @LaurentGoderre